Accessibility/Accessibility/Cells/CategoryCollectionViewCell.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  CategoryCollectionViewCell.swift
+//  Accessibility
+//
+//  Created by Nicole Souvenir on 1/23/19.
+//  Copyright © 2019 Nicole Souvenir. All rights reserved.
+//
+
+import UIKit
+import URBNSwiftyConvenience
+
+class CategoryCollectionViewCell: UICollectionViewCell {
+
+    private let imageView = UIImageView()
+    private let title = UILabel()
+    private let price = UILabel()
+    
+    private let marginOffset: CGFloat = 18.0
```

Comment:
This is fine for now, but we should probably add a card to create a `Margins` struct to easily have access to these values

Reply:
The margins were removed because this cell was refactored to a tableView cell .